client/src/components/TrackList.vue

```diff
@@ -132,14 +132,6 @@ export default Vue.extend({
         types: allTypes,
       };
     },
-
-    setType(trackId: TrackId, newType: string) {
-      const track = this.trackMap.get(trackId);
-      if (track === undefined) {
-        throw new Error(`Accessed missing track ${trackId}`);
-      }
-      track.setType(newType);
-    },
```

Comment:
I think for the purposes of the tool I would want to keep this and it's corresponding binding in the template.

I can use the `emit('track-type-change')` logic utilizing just the type instead of the Id for the CreationMode.vue (What I'm calling the component for the new track settings). I might change the name of it too to be something like 'default-new-track-type'